test/e2e/storage/testsuites/subpath.go

```diff
@@ -160,8 +160,8 @@ func (s *subPathTestSuite) DefineTests(driver storageframework.TestDriver, patte
 		e2epod.SetNodeSelection(&l.formatPod.Spec, l.config.ClientNodeSelection)
 
 		l.subPathDir = filepath.Join(volumePath, subPath)
-		l.filePathInSubpath = filepath.Join(volumePath, fileName)
-		l.filePathInVolume = filepath.Join(l.subPathDir, fileName)
```

Comment:
I touched the code last, but it already was like this before I got involved: 
https://github.com/kubernetes/kubernetes/blob/ba818d8dffd0f8c76471e05d931b20df6569fa2a/test/e2e/storage/subpath.go#L91-L92

I think @jsafrane was the original author.

The change seems to make sense, but I don't see a description (neither here nor in the issue) that explains what problem this change is fixing. Aren't tests passing with and without the change? Does that mean that we have a bigger problem with the test because it shouldn't be passing without this PR?

Reply:
@pohly The code is logically correct , but namings used  in parameters may sound confusing .

Reply:
If the code was correct before, why is then okay to initialize `l.subPathDir` differently now? It was `filepath.Join(volumePath, subPath)` before, now it is just `volumePath`.

If `l.subPathDir` is not used anywhere else, then remove it instead of arbitrarily changing what it contains.

Reply:
Hi @pohly  , I have looked into this again, the issue is about improving naming parameter for  `l.filePathInSubpath` and `l.filePathInVolume` so i tried swapping there parameters as 

```
l.filePathInVolume = filepath.Join(volumePath, fileName)
l.filePathInSubpath = filepath.Join(l.subPathDir, fileName)
```
But CI jobs are failing for the same changes too . So i have tried  tested initialise `volumePath` to `l.subPathDir` . It seems like this changes have further dependencies  . So should we close this PR?  or any thoughts to improve this .

Thanks!

Reply:
If it's not broken, don't fix it :grin:  I'm in favor of simply closing this PR.

If you want to improve the naming of variables, then those changes have to be made consistently everywhere. That doesn't seem to be the case right now (`l.subPathDir`!).

Reply:
@pohly Thanks for the clarification!